Shaders/Private/CesiumVoxelTemplate.usf

```diff
@@ -0,0 +1,224 @@
+// Copyright 2020-2025 CesiumGS, Inc. and Contributors
+
+/*=============================================================================
+	CesiumVoxelTemplate.usf: Template for creating custom shaders to style voxel data.
+=============================================================================*/
+
+// This depends on CesiumVoxelOctree.usf, but a local #include cannot be resolved here.
+// It must be linked under "Include File Paths" in a Custom node in the material.
+
+/*=======================
+  BEGIN CUSTOM SHADER
+=========================*/
+
+struct CustomShaderProperties
+{
+%s
+};
+
+struct CustomShader
+{
+%s
+  
+  float4 Shade(CustomShaderProperties Properties)
```

Comment:
Why not make this Shade function a member of `CustomShaderProperties` instead? Then the properties can be accessed without qualifying them with `Properties.`.